modules/qtwidgets/colorwheel.cpp

```diff
@@ -48,20 +48,19 @@
 #include <algorithm>
 #include <cmath>
 
-namespace inviwo {
-
-template <class T>
-inline T clamp(T value, T lower, T higher) {
-    return std::min(std::max(value, lower), higher);
+template <typename T>
```

Comment:
This should probably be in a anonymous namespace to avoid potential link issues...
Is also in the standard nowdays ;)